lib/base/strings.dart

```diff
@@ -0,0 +1,18 @@
+abstract class Strings {
+  static const appTitle = 'Cafe Analog';
+
+  // Titles for the app bar.
+  static const ticketsPageTitle = 'Tickets';
+  static const receiptsPageTitle = 'Receipts';
+  static const statsPageTitle = 'Statistics';
+  static const settingsPageTitle = 'Settings';
+
+  // Titles for the navigation bar.
+  static const ticketsNavTitle = ticketsPageTitle;
+  static const receiptsNavTitle = receiptsPageTitle;
```

Comment:
Minor comment - you don't have to change if it's by a good reason.. I'm not sure why you don't just use the page title where needed instead of creating a new constant?

Reply:
It's mostly for flexibility. If we were to implement another language, we may use shortened names for other pages 🙂